h/cli/commands/shell.py

```diff
@@ -1,83 +0,0 @@
-# pylint: disable=redefined-outer-name
-"""Command to run an interactive shell with application context."""
-
-import click
-
-BANNER = """Environment:
-  m, models    The `h.models` module.
-  registry     Active Pyramid registry.
-  request      Active request object.
-  session      Active database session.
-"""
-
-
-def autodetect():
-    try:
-        import bpython
-
-        return "bpython"
-    except ImportError:
-        try:
-            import IPython
-
-            return "ipython"
-        except ImportError:
-            pass
-
-    return "plain"
```

Comment:
`bpython` isn't in our requirements files, so it will never be installed. `ipython` is in our requirements, so it will always be installed. So unless this is for people who might want to manually hack their tox environments to install `bpython` (or uninstall `ipython`) this could be replaced with just `return "ipython"`.